contents/blog/brand.md

```diff
@@ -0,0 +1,128 @@
+---
+date: 2023-06-25
+title: How to brand your startup
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - james-hawkins
+category: CEO diaries
+---
+
+The world would be a lot more fun if most startups didn't seem to have experienced a personality vasectomy.
+
+I believe humans have evolved to conform - when we all lived in the wilderness, staying alive required you to fit in with a tribe, after all. Fast forward that background to today and, amongst other things (!), the majority of the world's software companies feel the same as each other.
```

Comment:
i think this very sweeping point, taking in all of human evolution, kinda weakens your argument. humans haven't evolved to conform, humans who create software companies tend to be somewhat similar humans solving somewhat similar problems... hence the uniformity. would suggest either cutting or reworking this to narrow down your point a bit